incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 568
+  runAsGroup: 568
+  fsGroup: 568
+
+timezone: "America/Chicago"
+
+env:
+  LATITUDE: 46
+  LONGITUDE: -94
+  ELEVATION: 1217
+  DASH_URL: "http://0.0.0.0:5050"
+
+services:
+  main:
+    port:
+      port: 5050
+  tcp:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 51050
+      protocol: TCP
+      targetPort: 51050
+ingress:
+  main:
+    enabled: false
+
+additionalVolumeMounts:
+  - name: config
+    mountPath: /conf
+
+
+additionalVolumes:
+  - name: config
+    hostPath:
+      path: /tmp
```

Comment:
Don't use other functions for storage as you normally would, it makes the tests inaccurate!
True something like this:
```
customStorage:
  - name: config
    enabled: true
    mountPath: /conf
    emptyDir: true
```